joern-cli/frontends/rubysrc2cpg/src/test/scala/io/joern/rubysrc2cpg/passes/ast/SimpleAstCreationPassTest.scala

```diff
@@ -1119,4 +1119,21 @@ class SimpleAstCreationPassTest extends RubyCode2CpgFixture {
     callNode.lineNumber shouldBe Some(2)
     callNode.columnNumber shouldBe Some(9)
   }
+
+  "have correct structure for body statements inside a do block" in {
```

Comment:
We already have this test [here](https://github.com/joernio/joern/pull/3366/files#diff-45b026c78f0405d131600a5e173a4dc262cdaae9ad115dfae83deb1ac6f1a938R1123), right?

Reply:
Ah, okay. This must be something with commits. We can close this PR.